### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by amon-cofie**
In this "To-Do-List" project, I created the basic structure of a page single page application with HTML, CSS and javascript, using the webpack bundler to bundle all the source files in the "src" folder into the "dist" folder for distribution. I did the following things:

- Installed the webpack bundler
- Generated the index.html file in the dist folder from the source folder using an html plugin
- Wrote functions function to iterate over the tasks array and populate an HTML list item element for each task.
- Wrote functions to add a task and remove the task
- Added CSS styling for the to do list application.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+
+resetBtn.addEventListener('click', () => {
+  localStorage.clear();
+  tasks.splice(tasks[0]);
+
+  console.log(tasks);
+  while (toDoList.firstChild) {
+    toDoList.removeChild(toDoList.lastChild);
+  }
+});
```
Comment (HAMMAS-SALEEM) [inline src/index.js]: req-missing-editing-task


- [ ] You have added add & remove functionality :clap: but you didn't add the functionality for editing an exisitng task which is a crucial requirement of this milestone :clap: Kindly implement the functionality for editing the task :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @amon-cofie :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitFlow
:heavy_check_mark: Following some requirements
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (amon-cofie) [reply src/index.js]: Thank you for pointing that out. I will implement that.

**Review by cwaku — APPROVED** [review]
Hi @amon-cofie

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**